### ISSUE TITLE
Add wiki documentation

### ISSUE THREAD

**Issue by matshou** [yooksi/Odyssey#11]
We should start documenting all mod mechanics in the repository wiki.

**Comment by yooksi** [comment]
Done, the [wiki](https://github.com/yooksi/Odyssey/wiki) tab should now be visible.

**Comment by yooksi** [comment]
@Titus-Armillius Could we get the day/night cycle documentation up on the wiki now that we've implemented that mechanic in our first pre-release?

**Comment by Titus-Armillius** [comment]
@yooksi I'll add the day/night cycle documentation asap.

Just to be sure; is the number value in the config a multiple? e.g. -12 = 12x slower?

**Comment by yooksi** [comment]
Yes, that is correct. See [TimeCycle javadoc](https://github.com/yooksi/Odyssey/blob/master/src/main/java/io/yooksi/odyssey/core/TimeCycle.java#L21) for more information.
